docs/usage.md

```diff
@@ -0,0 +1,86 @@
+# Usage
```

Comment:
Should we have an `About` section that talks at a high-level about what the import tool is used for?

Reply:
good idea, added